src/styles/blocks/catalog.scss

```
@@ -12,14 +12,21 @@
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
+    width: $cardWidth;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms ease,
+      color 300ms ease;
```

Create a variable for transition duration, it's important)
```suggestion
    margin-bottom: 48px;
    transition: transform $transitionDuration ease,
      color $transitionDuration ease;
```